shimmingtoolbox/shim/sequencer.py

```diff
@@ -1454,6 +1466,138 @@ def print_rt_metrics(self, unshimmed, shimmed_static, shimmed_static_riro, shimm
                      f"\nstatic_shim_riro: {static_shim_riro}"
                      f"\nstatic_unshimmed: {static_unshimmed}")
+    
+    def plot_full_mask(self, unshimmed, shimmed_masked, mask):
+        """
+        Plot and save the static full mask
+
+        Args:
+            unshimmed (np.ndarray): Original fieldmap not shimmed
+            shimmed_masked(np.ndarray): Masked shimmed fieldmap
+            mask (np.ndarray): Binary mask in the fieldmap space
+        """
+        # Plot
+     
+        mt_unshimmed = montage(unshimmed)
+        mt_unshimmed_masked = montage(unshimmed * mask)
+        mt_shimmed_masked = montage(shimmed_masked)
+        
+        metric_unshimmed_std = calculate_metric_within_mask(unshimmed, mask, metric='std')
+        metric_shimmed_std = calculate_metric_within_mask(shimmed_masked, mask, metric='std')
+        metric_unshimmed_mean = calculate_metric_within_mask(unshimmed, mask, metric='mean')
+        metric_shimmed_mean = calculate_metric_within_mask(shimmed_masked, mask, metric='mean')
+        metric_unshimmed_absmean = calculate_metric_within_mask(np.abs(unshimmed), mask, metric='mean')
+        metric_shimmed_absmean = calculate_metric_within_mask(np.abs(shimmed_masked), mask, metric='mean')
+
+        min_value = min(mt_unshimmed_masked.min(), mt_shimmed_masked.min())
+        max_value = max(mt_unshimmed_masked.max(), mt_shimmed_masked.max())
+
+        fig = Figure(figsize=(9, 6))
+        fig.suptitle(f"Fieldmaps\nFieldmap Coordinate System")
+
+        ax = fig.add_subplot(1, 2, 1)
+        ax.imshow(mt_unshimmed, cmap='gray')
+        mt_unshimmed_masked[mt_unshimmed_masked == 0] = np.nan
+        im = ax.imshow(mt_unshimmed_masked, vmin=min_value, vmax=max_value, cmap='viridis')
+        ax.set_title(f"Before shimming\nSTD: {metric_unshimmed_std:.3}, mean: {metric_unshimmed_mean:.3}, "
+                     f"abs mean: {metric_unshimmed_absmean:.3}")
+        ax.get_xaxis().set_visible(False)
+        ax.get_yaxis().set_visible(False)
+        divider = make_axes_locatable(ax)
+        cax = divider.append_axes('right', size='5%', pad=0.05)
+        fig.colorbar(im, cax=cax)
+
+        ax = fig.add_subplot(1, 2, 2)
+        ax.imshow(mt_unshimmed, cmap='gray')
+        mt_shimmed_masked[mt_shimmed_masked == 0] = np.nan
+        im = ax.imshow(mt_shimmed_masked, vmin=min_value, vmax=max_value, cmap='viridis')
+        ax.set_title(f"After shimming\nSTD: {metric_shimmed_std:.3}, mean: {metric_shimmed_mean:.3}, "
+                     f"abs mean: {metric_shimmed_absmean:.3}")
+        ax.get_xaxis().set_visible(False)
+        ax.get_yaxis().set_visible(False)
+        divider = make_axes_locatable(ax)
+        cax = divider.append_axes('right', size='5%', pad=0.05)
+        fig.colorbar(im, cax=cax)
+
+        # Lower suptitle
+        fig.subplots_adjust(top=0.85)
+
+        # Save
+        print("---------------------------------------------------------------- \n SAVING")
```

Comment:
```suggestion
```

If you want to print something, use `logger.info("")`

Reply:
However in this case I would not output anything since there could be multiple plots that could output "saving".